homeassistant/helpers/config_validation.py

```diff
@@ -1119,6 +1126,26 @@ def key_value_validator(value: Any) -> dict[Hashable, Any]:
     return key_value_validator
 
 
+def get_absolute_description_key(domain: str, key: str) -> str:
```

Comment:
I am not sure if `config_validation` is the right place for this. 
also, `get_absolute_description_key` in the context of config is really generic and not clear about what it is.
maybe a new `automation.py` file in `util` instead?